snippets/provider_listenable.code-snippets

```diff
@@ -0,0 +1,8 @@
+{
+  "Provider Listenable": {
+    "scope": "dart",
+    "prefix": "listen",
+    "description": "Create a provider listenable",
+    "body": ["ref.listen<$1>($2, (value) {", "});"]
```

Comment:
This still looks a bit weird, specifically the `{". "}`

Either make it an arrow function or separate it in to multiple lines

Reply:
Sure